RELEASE_NOTES.rst

```diff
@@ -4,6 +4,7 @@ Next release
 All changes
 -----------
 
+- :pull:`460`: Expand documentation :doc:`INSTALL` for installing GAMS under macOS.
```

Comment:
Can you double-check that this link works when the docs are built? Although the file is at the project root in INSTALL.rst, it's included in the documentation via doc/install.rst, so I suspect the lower case is needed within this link.

Reply:
Thanks @khaeru. I have corrected and checked that the link works.